docs/ossec_alerts.rst

```diff
@@ -90,39 +90,44 @@ install\_files/ansible-base and then specify the filename in the
 If you don't have your GPG key ready, you can run GnuPG on the command
 line in order to find, import and export your public key. It's best to
 copy the key from a trusted and verified source, but you can also
-request it from keyservers using the known longid. Looking it up by
-email address could cause you to obtain a wrong, malicious or expired
-key. Just replace the key ID in the following example with yours:
+request it from keyservers using the known fingerprint. Looking it up by
+email address or a shorter key ID format could cause you to obtain a
+wrong, malicious or expired key. Just replace the word ``<fingerprint>``
+in the following example with your own:
 
-Download your key and import it into the local keyring:
+Download your key and import it into the local keyring: ::
 
-::
+    gpg --keyserver <fingerprint> hkp://qdigse2yzvuglcix.onion --recv-key C5986B4F1257FFA86632CBA746181433FBB75451
```

Comment:
The previous paragraph says "replace the word `<fingerprint>` in the following example with your own", but then the example has a specific fingerprint (`C5986B4F1257FFA86632CBA746181433FBB75451`) instead of `<fingerprint>`. I would prefer `<fingerprint>` to avoid potential confusion.